README.md

```diff
@@ -35,14 +35,18 @@ The following table lists parameters to be set.
 
 | Parameter | Description |
 | ---- | ---- |
-| appid | App ID, which is obtained from app information. |
-| appsecret | Secret access key of an app, which is obtained from app information. |
+| appid | The App ID of the app receiving the push messages. The App ID can be obtained from the App information section in AppGallery. |
```

Comment:
'of the app' is a bit redundant, because it's assumed the AppID is about the app itself